mod/primitives/pkg/encoding/hex/bytes.go

```
@@ -39,7 +40,13 @@
 
 // MustToBytes returns the bytes represented by the given hex string.
 // It panics if the input is not a valid hex string.
-func MustToBytes(input string) []byte {
+func MustToBytes(input string) (bz []byte) {
```

golangci-lint (1.62.2)

named return "bz" with type "[]byte" found

(nonamedreturns)